src/Components/Semester/Semester.module.scss

```diff
@@ -1,6 +1,6 @@
 .container {
   height: 100vh;
-  width: 100vw;
+  width: 98vw;
```

Comment:
Changing 100vw to 98vw for removing horizontal scrollbar is absurd. Use 100% for width.

Please google stuff like this before using maths to accomplish it, as that maths will most likely only work on your PC.